index.html

```diff
@@ -1,23 +1,91 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
     <title>Document</title>
-</head>
-<body>
-    <section class="header-nav">
-        <div class="container">
-            <img src="images/↳ Time.png" alt="time">
-            <img src="images/vec.svg" alt="time" id="capacity">
-            <img src="images/Cap.png" alt="time" id="cap">
-
-
+  </head>
+  <header>
+    <nav class="status-bar">
+      <p class="hour">9:41</p>
+      <div class="icons">
+        <div class="network">
+          <img src="images/net.svg" alt="connection" />
         </div>
-        
+        <div class="wifi">
+          <img src="images/wifi.svg" alt="wifi icon" />
+        </div>
+        <div class="battery">
+          <img src="images/vec.svg" alt="battery icon" />
+        </div>
+      </div>
+      </nav>
+    <nav class="menu">
+      <h6 class="name">Kiprotich Kimutai</h6>
+      <button
+        type="button"
+        aria-label="open close main menu"
+        class="toolbar-icon"
+      >
+        <img src="images/menu.png" alt="" />
```

Comment:
- [ ] [OPTIONAL] As pointed out by the previous reviewer, please add an alt text to make site more accessible to people with disabilities that use screen readers.